suite-native/alerts/src/useAlert.ts

```diff
@@ -0,0 +1,26 @@
+import { useAtom, atom } from 'jotai';
+import { RequireAllOrNone } from 'type-fest';
+
+export type AlertType = 'network-error';
+export type Alert = RequireAllOrNone<
+    {
+        title: string;
+        description: string;
+        primaryButtonTitle: string;
+        onPressPrimaryButton: () => void;
+        secondaryButtonTitle?: string;
+        onPressSecondaryButton?: () => void;
+    },
+    'secondaryButtonTitle' | 'onPressSecondaryButton'
+>;
+
+const alertAtom = atom<Alert | null>(null);
+
+export const useAlert = () => {
+    const [alert, setAlert] = useAtom(alertAtom);
+
+    return {
+        alert,
+        setAlert,
```

Comment:
nit: I would prefer `showAlert`

Reply:
if the parameter of this function would be `boolean` I would agree. But since the parameter is `Alert` object or null, setAlert makes better sense to me. IMHO it is better to stick to the same convention which is used for `useState` hook.